### ISSUE TITLE
broot blocking ctrl-z (SIGSTOP)

### ISSUE THREAD

**Issue by errnoh** [Canop/broot#735]
Version: `broot 1.22.0`

First of all, thank you for a great software that instantly became central part of how I navigate my systems.
That said, there's one thing that keeps nagging me. For some reason broot blocks SIGSTOP (well, it actually blocks ctrl-z keypress, more on that later), which something I tend to constantly use to avoid opening new terminal windows for everything.

I guess part of the reason to do this is hinted by sending SIGSTOP to the process manually: broot suspends as expect, but... when moving mouse in the window it was running in it keeps spamming special codes on the terminal. Resuming the process with `fg` also didn't instantly return to the file listing, but required additional press of enter (possibly some other key might've worked).

tl;dr: broot blocks `ctrl-z`, but fixing that would likely also require spending time handling `SIGSTOP` in a clean way.

**Comment by Canop** [comment]
Are you using `ctrl-z` because you'd like broot to stay unchanged while you do a short operation on the terminal ?

If so you may also use `ctrl-t`, which opens a new shell (on exit you're back to broot).

**Comment by errnoh** [comment]
@Canop Thanks, that does seem to be viable workaround for the issue! Support for the native solution would be nice, but this at least removes the main annoyance I was having and I understand that it would likely be a non-trivial effort for minimal user value so I'm ok with considering this issue complete.
